web/models/terms/term.ex

```diff
@@ -31,5 +32,17 @@ defmodule CoursePlanner.Terms.Term do
     |> validate_required([:name, :start_date, :end_date, :status])
     |> cast_embed(:holidays)
     |> Statuses.update_status_timestamp(EntityStatus)
+    |> validate_date_range()
   end
+
+  defp validate_date_range(%{errors: []} = changeset) do
```

Comment:
I think it is more usual to check the for `%{valid?: true}` in this case, can you take a look?